template.yaml

```diff
@@ -1435,6 +1435,9 @@ Outputs:
     Export:
       Name: !Sub DocAppApiGateway-${Environment}
   PerfTestApiId:
-    Value: !Ref PerfTestApiGateway
+    Value: !If
+      - IsStaging
+      - !Ref PerfTestApiGateway
+      - "no-value"
```

Comment:
nitpick: I think elsewhere we've been using `AWS::NoValue` - any reason not to go with this?